File: task-03/hello world/diamond/diamond.rb
n = gets.to_i

(0...n).each do |i|
  puts " " * (n - i - 1) + "*" * (2 * i + 1)
end

(n - 2).downto(0) do |i|
  puts " " * (n - i - 1) + "*" * (2 * i + 1)
end

File: task-03/hello world/diamond/diamond.ex
n = String.to_integer(IO.gets("Enter a number: "))

for i <- 0..(n - 1) do
  IO.puts String.duplicate(" ", n - i - 1) <> String.duplicate("*", 2 * i + 1)
end

for i <- (n - 2)..0 do
  IO.puts String.duplicate(" ", n - i - 1) <> String.duplicate("*", 2 * i + 1)
end
